### ISSUE TITLE
how to include many and legacy files

### ISSUE THREAD

**Issue by Moriand** [pyinstaller/pyinstaller#3527]
Hello, I am currently developing a GUI in PyQT5 "eye -> I am a student"
I have several py files:
1) **program.py**
2) **content.py**

**1) program.py**

This contains some additions such as actions by pressing a button as well as the import of the icon.

**2) content.py**

Here is where the graphic part of the program is, this one had created it with PyQt5 designer, and converted to py with pyuic5.

when I link with python:
`pyinstaller program.py`

it is an error and I have also tried creating a p file that links to both p and p files ..

link.py
contains
```
import program.py
import contenido.py
```

**Comment by tallforasmurf** [comment]
You must give MUCH more information before anyone can help you. What OS? What versions of PyInstaller, Python, PyQt, the OS? What kind of error? Show the error messages.

I recommend you close this and start a discussion with the above info on the PyInstaller mailing list,
      https://groups.google.com/forum/#!forum/pyinstaller
Then people can ask questions and help you.

**Comment by htgoebel** [comment]
PyInstaller should do, what you expect. But:

> link.py
contains
```
import program.py
import contenido.py
```

Do be frank: I suggest you to learn Python prior to trying to use PyInstaller. Additionally PyQt5 is a beast which PyInstaller is still struggling with.

**Comment by Moriand** [comment]
[Resolved] windows 8.1
the best way is if you have installed in **C: / program files x64 / Python35**
uninstall it completely and reinstall python at the following address

**C: / Python35**
this problem is because Pyinstaller does not differentiate the spaces of the python packaging

**Comment by htgoebel** [comment]
> this problem is because Pyinstaller does not differentiate the spaces of the python packaging

I doubt this statement is correct.
